features/exploredash/src/main/java/org/dash/wallet/features/exploredash/data/model/merchent/GetDataMerchantIdResponse.kt

```diff
@@ -489,13 +489,13 @@ data class GetDataMerchantIdResponse(
             @SerializedName("LogoUrl")
             val logoUrl: String? = "",
             @SerializedName("MaximumCardPurchase")
-            val maximumCardPurchase: Int? = 0,
+            val maximumCardPurchase: Double? = 0.0,
             @SerializedName("MinimumCardPurchase")
-            val minimumCardPurchase: Int? = 0,
+            val minimumCardPurchase: Double? = 0.0,
             @SerializedName("PaymentInstructions")
             val paymentInstructions: String? = "",
             @SerializedName("SavingsPercentage")
-            val savingsPercentage: Int? = 0,
+            val savingsPercentage: Double? = 0.0,
```

Comment:
This file is not currently being used -- this is the class that contains all fields returned, though we are only using 3 of them.